src/components/TradeWidget/OrderDetails.tsx

```diff
@@ -4,6 +4,8 @@ import styled from 'styled-components'
 import { FEE_PERCENTAGE } from 'const'
 import Highlight from 'components/Highlight'
 
+const DECIMALS_FOR_PRICE = 4
```

Comment:
can you use the `const` version or put this inside `const`?

Reply:
I should, but this disappears in next PR: https://github.com/gnosis/dex-react/pull/398